ui/catalog_selection.ui

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file looks like it accidentally leaked in to your commit?

Reply:
Actually this one not, it's the .ui for the ui_catalog_selection.py under gui/bluesky. The treeview_test.ui should not be in there yet...